build/docker/kmesh.dockerfile

```diff
@@ -18,9 +18,10 @@ COPY --from=builder /usr/lib64/libkmesh_api_v2_c.so /usr/lib64/
 COPY --from=builder /usr/lib64/libkmesh_deserial.so /usr/lib64/
 COPY --from=builder /usr/lib64/libboundscheck.so /usr/lib64/
 COPY --from=builder /kmesh/*.ko /kmesh
-COPY --from=builder /kmesh/oncn-mda/build/ebpf_src/CMakeFiles/sock_redirect.dir/sock_redirect.c.o /usr/share/oncn-mda/
+COPY --from=builder /kmesh/out/*.o /usr/share/oncn-mda/

```

Comment:
The path is not right

``
should be /kmesh/oncn-mda/build/ebpf_src/CMakeFiles/sock_redirect.dir/sock_redirect.c.o

Reply:
and add `kmesh/oncn-mda/build/ebpf_src/CMakeFiles/sock_ops.dir/sock_ops.c.`